components/n-ui/ads/index.js

```diff
@@ -66,20 +46,6 @@ function onAdsComplete (flags, event) {
 
 		if (detail.slot.gpt && detail.slot.gpt.isEmpty === false) {
 			utils.log.info('Ad loaded in slot', event);
-			if (slotsRendered === 0) {
```

Comment:
Where has all this code moved to? It looks like the `onAdsComplete` function is now just logging things...should we remove it? Or add the logging to wherever this logic has moved to?

Reply:
We are not doing anything special with the 'first' slot rendered now. We treat like any other slot.
I noticed this was only logging but it was not part of my work to remove this logging functionality. 
I am happy to remove it as part of a separate PR. @andrewgeorgiou1981, @SamiTriki @alevito what are your thoughts?

Reply:
Makes sense. I'll add a card on trello for this